pkgs/development/ocaml-modules/ipaddr/default.nix

```diff
@@ -1,20 +1,24 @@
-{ stdenv, fetchurl, buildDunePackage, sexplib, ppx_sexp_conv }:
+{ stdenv, fetchurl, buildDunePackage, ocaml
+, macaddr, ounit, sexplib
+}:
 
 buildDunePackage rec {
   pname = "ipaddr";
-  version = "2.8.0";
+  version = "3.1.0";
 
   src = fetchurl {
-    url = "https://github.com/mirage/ocaml-${pname}/archive/${version}.tar.gz";
-    sha256 = "1amb1pbm9ybpxy6190qygpj6nmbzzs2r6vx4xh5r6v89szx9rfxw";
+    url = "https://github.com/mirage/ocaml-${pname}/archive/v${version}.tar.gz";
+    sha256 = "1hi3v5dzg6h4qb268ch3h6v61gsc8bv21ajhb35z37v5nsdmyzbh";
   };
 
-  propagatedBuildInputs = [ ppx_sexp_conv sexplib ];
+  buildInputs = [ macaddr ounit sexplib ];
```

Comment:
Why is `sexplib` needed?

Reply:
It actually isn't - see the reply to the same question in `macaddr`.